client/client.Dockerfile

```diff
@@ -49,22 +29,20 @@ RUN pip3 install \
       --find-links=/root/wheels \
       -r requirements.txt
 
-# COPY --from=gitclone /enigma-contract/enigma-js /root/enigma-contract/enigma-js
-COPY enigma-js/package.json /root/enigma-contract/enigma-js/
-COPY enigma-js/.babelrc /root/enigma-contract/enigma-js/
-WORKDIR /root/enigma-contract/enigma-js
 
-RUN yarn install
+COPY --from=gitclone_integration /integration-tests /root/integration-tests
+COPY --from=gitclone_contract /enigma-contract/enigma-js /root/integration-tests/enigma-js
```

Comment:
I would push back on this one. Why download again something that we should already have locally? It's only a matter of making sure that the `contract` image is built before this one.

Reply:
While I don't agree that someone would have the contract image locally, I don't necessarily mind. I really just want to avoid the confusion of someone that isn't familiar with what's going on when the build fails. 
How about adding the gitclone of contract to the make of the client?

Reply:
Added in 77dae288e58d620947fa0af7267d93246dba2e28